Atomic/util.py

```diff
@@ -62,7 +62,7 @@ def decompose(compound_name):
         reg, repo = repo.split('/', 1)
     if ':' in repo:
         repo, tag = repo.rsplit(':', 1)
-    return reg, repo, tag
+    return str(reg), str(repo), str(tag)
```

Comment:
No I just wanted to make sure you tested it on both pythons.  Code like this can sometimes trigger other issues.